src/components/alert/alert.ts

```diff
@@ -17,7 +17,7 @@ export class Alert extends ViewController {
   constructor(app: App, opts: AlertOptions = {}, config: Config) {
     opts.inputs = opts.inputs || [];
     opts.buttons = opts.buttons || [];
-    opts.enableBackdropDismiss = isPresent(opts.enableBackdropDismiss) ? !!opts.enableBackdropDismiss : true;
+    opts.enableBackdropDismiss = isPresent(opts.enableBackdropDismiss) ? Boolean(opts.enableBackdropDismiss) : true;
```

Comment:
No chance I convince you it is ugly, unnecessary operation, funny code, and does not show it's intent? (it took me a couple of minutes to figure out this is how you cast)